scripts/reference.py

```
@@ -0,0 +1,448 @@
+            pass
+        out.append(_conf_to_atoms(m, conf.GetId()))
+    return out
+
+
+def _read_trail(path):
+    """Read a geometry-trail XYZ; returns [(atoms, e_e0_eV, kind)] in file order."""
+    from ase.io import read
+    frames = read(path, index=":")
+    out = []
+    comments = [ln.strip() for ln in open(path) if "kind=" in ln]
+    for atoms, c in zip(frames, comments):
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)